cg/constants/constants.py

```diff
@@ -57,7 +57,7 @@ class ControlOptions(StrEnum):
     EMPTY: str = ""
 
 
-DEFAULT_CAPTURE_KIT = "twistexomerefseq_9.1_hg19_design.bed"
+DEFAULT_CAPTURE_KIT = "twistexomerefseq_10.2_hg19_design.bed"
```

Comment:
Out of scope, but in the future we should store this in a database and fetch it when needed.